pom.xml

```diff
@@ -5,27 +5,86 @@
 
   <groupId>com.google.cloud.tools.ide.login</groupId>
   <artifactId>google-ide-login</artifactId>
-  <version>0.0.0-SNAPSHOT</version>
+  <version>0.1.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
-  <name>Login Support for Google Cloud Tools</name>
+  <name>IDE Login Support for Google Cloud Tools</name>
```

Comment:
I'm not sure. Easier to do now than later, but they are really just arbitrary strings. com.google.cloud.tools.ide.login we should keep since that's the package name.